PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -103,7 +104,13 @@ if (!matchingInputReportFiles || matchingInputReportFiles.length === 0) {
     tl.warning('No test result files matching ' + inputReportFiles + ' were found.');
     tl.setResult(tl.TaskResult.Succeeded, '');
 } else {
-    transformReports(matchingInputReportFiles, 0);
+    if (isDtpSettingsValid) {
+        verifyDTPService().then(() => {
```

Comment:
What would happen now if the settings file is provided with valid properties, but the service is not availabe?
I believe we should still transform the report right?

Reply:
Yes, regardless of whether the DTP service is availabe or not, we should still transform the report.

Reply:
It reachs then block even DTP service is not availabe after calling verifyDTPService() method, since verifyDTPService() method catchs the error and return the resolved status.